src/libostree/ostree-repo.c

```diff
@@ -3292,6 +3292,35 @@ ostree_repo_get_mode (OstreeRepo  *self)
   return self->mode;
 }
 
+/**
+ * ostree_repo_get_min_free_space:
+ * @self: Repo
+ * @error: Return location for a #GError
+ *
+ * It should be noted that this function should be used only if there
+ * is a transaction active. It is a programmer error to request it
+ * otherwise.
+ *
+ * If the configuration value of min-free-space-* value is too high(overflow)
+ * then @error is set with status code %G_IO_ERROR_FAILED.
+ *
+ * Returns: Value(in bytes) of min-free-space-* config option
+ * Since: 2018.9
+ */
+guint64
+ostree_repo_get_min_free_space_bytes (OstreeRepo  *self, GError **error)
+{
+  g_return_val_if_fail (OSTREE_IS_REPO (self), 0);
+  g_return_val_if_fail (error == NULL || *error == NULL, 0);
+  g_return_val_if_fail (self->in_transaction == TRUE, 0);
+
+  g_assert (self->txn.blocksize != 0);
+  if (self->reserved_blocks > G_MAXUINT64 / self->txn.blocksize)
+    return glnx_throw (error, "min-free-space(in bytes) value was too high");
```

Comment:
(Make sure this error message is the same as whatever you change the one above to be.)